package.json

```diff
@@ -30,6 +30,30 @@
     "./src/*.js": "./src/*.js"
   },
   "main": "dist/jquery.js",
+  "scripts": {
+    "build": "npm install && npm run build-all-variants",
```

Comment:
Is there a good reason to move it all up? That breaks git history for selection.

Reply:
Not a *good* reason, but maybe an okay reason. We're going to be adding more scripts and working with them more often so I figured it'd be nice to have it closer to the top.

Reply:
Hmm, I see. I guess it's a good moment then as this PR doesn't change that much in scripts and further changes may be easier to review.